dev/benchmarks/test_apps/stocks/README.md

```diff
@@ -30,9 +30,9 @@ the tool is used to generate localizations for this app.
 
 ## Icon
 
-Icon was created using Android Asset Studio:
+The icon was created using Android Asset Studio:
 https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=image&foreground.space.trim=0&foreground.space.pad=0&foreColor=607d8b%2C0&crop=0&backgroundShape=square&backColor=fff%2C100&effects=none
 
 From this clipart:
 https://openclipart.org/detail/30403/tango-go-up
-Which is public domain.
+Which is the public domain.
```

Comment:
The clip art library is not "the public domain", it is either "public domain", or more explicitly "in the public domain".

```suggestion
Which is public domain.
```

Reply:
Okay 👍